load/deploy_test.go

```diff
@@ -0,0 +1,128 @@
+package load_test
```

Comment:
will this package naming overlap when load topology is added?

Reply:
No, I plan to add the topology code into this same package.